### PR TITLE
Propagate installers in runtime output in VMR build

### PR DESCRIPTION
**Pull request by mmitche**
Also, rename SourceBuild.props to DotNetBuild.props to reflect its purpose.

### CODE REVIEW

**Comment by mmitche** [comment]
Testing this in a full VMR build right now.

File: eng/DotNetBuild.props [hunk 1]
```diff
@@ -92,6 +91,15 @@
         Include="$(CurrentRepoSourceBuildArtifactsPackagesDir)Shipping\dotnet-crossgen2-*$(ArchiveExtension)"
         Category="Crossgen2Archive" />
     </ItemGroup>
+
+    <ItemGroup Condition="'$(DotNetBuildOrchestrator)' == 'true'">
```
[the remark this answers is not in the document]
Comment (ViktorHofer) [reply eng/DotNetBuild.props:95]: Why can't we use `DotNetBuild`?

[the remark this answers is not in the document]
Comment (mmitche) [reply eng/DotNetBuild.props:95]: We don't want to accidentally include these in the intermediate nupkgs that are necessary for source-build repo legs. I don't think that they're actually produced for those legs, but this is to make sure. If we do, include a .deb for instance, we might hit the AzDO package size limit.

They're only necessary when installer (and probably consumes) upstream installers for building its own installers.

[the remark this answers is not in the document]
Comment (mmitche) [reply eng/DotNetBuild.props:95]: The reason you can't use DotNetBuild is that it's active in a repo-only build.

**Review by ViktorHofer — APPROVED** [review]

**Pull request merged**